model/election.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, election_records):
         # Compose election slugs
         for el in elections:
             election_date = datetime.strptime(el['date'], '%Y-%m-%d')
-            ordinal_day = self.ordinal(int(election_date.strftime('%-d')))
```

Comment:
This is to the get the day of the month with the leading zero trimmed. I'm aware that it's some "secret" functionality of *nix systems and may not work everywhere, such as on Windows. Did it give you issues?

Reply:
yes, it didn't work for me.  Maybe it was the python version.  I can look further into getting it to do what you're trying to do.

Reply:
Looks like it's a windows issue: https://www.edureka.co/community/34421/python-strftime-date-without-leading-0#:~:text=If%20you%20add%20a%20hyphen,Y%2F%25%23m%2F%25%23d.  I'll test it on linux.

Reply:
reverted

Reply:
Are you on Windows? PEC staff is also on Windows. Just in case, say, their intern contributes to this project in the future, it could be worth fixing to work on all systems. What do you think?

Reply:
Another approach is to set up for codespaces and dev containers.  This will protect against future incompatibilities.  In other words, it's linux even on Windows.